streaming/src/main/scala/org/apache/gearpump/streaming/source/Watermark.scala

```diff
@@ -30,7 +30,7 @@ case class Watermark(instant: Instant) {
 
 object Watermark {
 
-  val MAX: Instant = Instant.ofEpochMilli(MAX_TIME_MILLIS)
+  val MAX: Instant = Instant.ofEpochMilli(MAX_TIME_MILLIS + 1)
```

Comment:
Watermark means no more messages with timestamps **less than** the watermark time should be seen. Hence, the maximum watermark time should be 1ms larger than maximum timestamp